src/routes/devices/deviceValidator.ts

```diff
@@ -10,9 +10,19 @@ export const metadataQueryValidator = (): any => {
     check('tags')
       .optional()
       .isString(),
+    check('status')
+      .optional()
+      .isNumeric,
```

Comment:
does this work? I would think this needs to be a function call like the rest? Additionally you should be able to have the validator convert it so you don't need to parseInt later.

Reply:
yes it does down.. doesn't work without the parseInt